src/Common/src/System/Net/Logging/EmailEventSource.cs

```diff
@@ -10,18 +10,18 @@
 
 namespace System.Net
 {
-    [EventSource(Name = "Microsoft-System-Net-Mail", LocalizationResources = "FxResources.System.Net.Mail.SR")]
-    internal sealed class WebEventSource : EventSource
+    [EventSource(Name = "Microsoft-System-Net-Mail", Guid = "32663b9f-4d8d-44f8-9f29-f4efc12e6879", LocalizationResources = "FxResources.System.Net.Mail.SR")]
```

Comment:
Even we chose not to localize the resources, we place all strings in the resource file anyways.  This is better than putting hard-coded strings in the *.CS files themselves.

So, I would prefer we keep this design as-is and leave strings in the resources file.